src/test/java/com/wikia/webdriver/elements/mercury/old/PortableInfoboxObject.java

```diff
@@ -337,22 +337,19 @@ public PortableInfoboxObject areDataValuesVisible() {
     return this;
   }
 
-  public PortableInfoboxObject areUnorderedListAndDataValuesMarginEqual() {
-    Assertion.assertEquals(
-        unorderedLists.get(0).getCssValue("margin"),
-        dataValues.get(0).getCssValue("margin")
-    );
-    PageObjectLogging.log("Unordered list labels and value", "have the same margin", true);
+  public PortableInfoboxObject verifyDataValueMargin() {
+    Assertion.assertEquals(dataValues.get(0).getCssValue("margin"), "0px");
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "margin" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1192)